src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import App from './App'
-import './global.css'
+import App from './components/App'
+import './components/global.css'
```

Comment:
This is not a component, but a css-file. In comparison: styled-component's `GlobalStyle.js` is a component. You can see it in the `App`, where you write `<GlobalStyle />` to use it.

Reply:
Ups, true. That was by mistake. Thanks for pointing it out!